Sources/Misc/DateAndTime/Clock.swift

```diff
@@ -13,21 +13,31 @@
 
 import Foundation
 
-/// A type that can provide the current `Date`
-protocol ClockType: Sendable {
+/// A type that can provide the current `Date` and `DispatchTime`.
+///
+/// - Note: Internal use only
```

Comment:
For the `@_spi(Internal)` APIs, I prefer to add the `// swiftlint:disable:next missing_docs` marker, as these are not really public APIs